backend/lib/services/projects.js

```diff
@@ -138,6 +138,14 @@ exports.list = async function ({ user, qs = {} }) {
     .gte(0)
     .value()
 
+  const isGroupMemberOf = project => !_
```

Comment:
I would change the isMemberOf function in a way that it also considers groups. The user is member of the project if one of his groups is member of the project. 

```js
const predicate = members => {
  const hasGroupMembership = _
    .chain(members)
    .filter(['kind', 'Group'])
    .map('name')
    .intersection(user.groups)
    .size()
    .gt(0)
    .value()
  const hasUserMembership = _
    .chain(members)
    .filter(['kind', 'User'])
    .map('name')
    .includes(user.id)
    .value()
  return hasGroupMembership || hasUserMembership
}